db/schema.rb

```diff
@@ -555,6 +556,16 @@
     t.index ["user_id"], name: "index_identities_on_user_id"
   end
 
+  create_table "instance_moderation_notes", force: :cascade do |t|
+    t.string "domain", null: false
+    t.bigint "account_id", null: false
```

Comment:
Rather than `ON DELETE CASCADE` we probably want to do like on delete set null or on delete set account_id to instance actor id (`-99`), otherwise a moderator deleting their account would remove information that would've been useful for other moderators in the future.

Reply:
Yes it might make sense to open an issue. Definitely not in scope for this particular PR though.

Reply:
Opened: https://github.com/mastodon/mastodon/issues/35169